overrides/partials/integrations/analytics/custom.html

```diff
@@ -1 +1,10 @@
-<script defer data-domain="docs.atlasos.net" src="https://insights.tweakcentral.net/js/script.outbound-links.file-downloads.js"></script>
\ No newline at end of file
+<!-- Plausible -->
+<script defer data-domain="docs.atlasos.net" src="https://insights.tweakcentral.net/js/script.outbound-links.file-downloads.js"></script>
+<!-- Microsoft Clarity -->
```

Comment:
How privacy impactful is this?

Reply:
[Plausible](https://plausible.io/) is privacy respecting as it doesn't track users between sessions. Clarity claims to not track users between sessions unless we collect consent, which we don't, as we don't want that cross-session tracking.

Reply:
Ah okay, that's fine then, as long as it isn't somehow tracking users in an invasive way.